launch/tier4_perception_launch/launch/perception.launch.xml

```diff
@@ -36,6 +36,7 @@
   <arg name="image_raw7" default="/sensing/camera/camera7/image_rect_color"/>
   <arg name="camera_info7" default="/sensing/camera/camera7/camera_info"/>
   <arg name="image_number" default="6" description="choose image raw number(1-8)"/>
+  <arg name="object_recognition_detection_fusion_sync_param_path" description="you need change the size of 'input_offset_ms' in the file in this path when you want to change image_number"/>
```

Comment:
The name is a bit long, so how about just 'fusion_roi_sync_param_path'
```suggestion
  <arg name="fusion_roi_sync_param_path" description="you need change the size of 'input_offset_ms' in the file in this path when you want to change image_number"/>
```

Reply:
https://github.com/autowarefoundation/autoware_launch/pull/350#discussion_r1199889738